helm/pmm-server/templates/statefulset.yaml

```diff
@@ -40,7 +40,7 @@ spec:
           ports:
             - name: https
               protocol: TCP
-              containerPort: 443
+              containerPort: 80
```

Comment:
Name is left `https` and port is `80`. Also could you explain why this change is needed?

Reply:
the name should be http to match 80. I guess the only part needed here was the